### PR TITLE
Add Contact Us page

### LINKED ISSUE #66: Add contact us page
Refer slide 36

### PR DESCRIPTION
**Pull request by himanshuvarandani**
Closes #66

Sending mail using [email.js](https://www.emailjs.com/).
It requires an account of [email.js](https://www.emailjs.com/), check [this](https://wallis.dev/blog/using-emailjs-with-nextjs) blog for account creation and giving parameters in ".env.local"

Example for .env:

EMAILJS_SERVICE_ID=new_service_id
EMAILJS_TEMPLATE_ID=new_template_id
EMAILJS_USER_ID=new_user_id

Screenshots:

For showing error,
image

For success,
image

For mail template,
image

### CODE REVIEW

**Comment by pnijhara** [comment]
@roshnet I have added the environment variable. Please merge this PR if everything is good. Thanks.

File: components/Header.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ export default function Header() {
               News &amp; Events
             </a>
           </Link>
-          <Link href="/">
+          <Link href="/contactUs">
```
Comment (pnijhara) [inline components/Header.js:29]: @himanshuvarandani I forgot to mention this.
This should be `<Link href="/contact">`

Comment (himanshuvarandani) [reply components/Header.js:29]: ok I will update this in another pr

Comment (pnijhara) [reply components/Header.js:29]: I think the email validation commit made was supposed to make this change.

Comment (pnijhara) [reply components/Header.js:29]: I am making a direct commit.

**Pull request merged**